src/solver/optim-model-filler/TimeDependentLinearExpression.cpp

```diff
@@ -91,13 +91,13 @@ TimeDependentLinearExpression BinaryOpLinearExpression(const LinearExpressionMap
     auto result(left);
     for (const auto& [timeStep, other_linear_expression]: right)
     {
-        if (result.contains(timeStep))
+        if (auto it = result.find(timeStep); it == result.end())
         {
-            result[timeStep] = op(result.at(timeStep), other_linear_expression);
+            result.emplace(timeStep, other_linear_expression);
         }
         else
         {
-            result[timeStep] = other_linear_expression;
+            it->second = op(it->second, other_linear_expression);
         }
```

Comment:
Not in the scope of this PR, but I don't think `a-b` is well defined when `a.timeSteps != b.timeSteps`